patches/net/minecraft/client/resources/model/ModelResourceLocation.java.patch

```diff
@@ -0,0 +1,21 @@
+--- a/net/minecraft/client/resources/model/ModelResourceLocation.java
++++ b/net/minecraft/client/resources/model/ModelResourceLocation.java
+@@ -8,6 +_,7 @@
+ @OnlyIn(Dist.CLIENT)
+ public record ModelResourceLocation(ResourceLocation id, String variant) {
+     public static final String INVENTORY_VARIANT = "inventory";
++    public static final String NEOFORGE_SPECIAL_VARIANT = "neoforge_special";
```

Comment:
I'm not a fan of the name. "Special" is not a very descriptive name, and the NeoForge prefix is not great either. Thoughts on `"standalone"`?

Reply:
Shouldn't the ID namespace be enough to avoid conflicts already?

Reply:
Right now, yes; we don't know what vanilla will do down the line however. It's up to you; I'm not picky.

Reply:
I think that we shouldn't worry about what vanilla will do in the future, and rather deliver the best possible API now.

So I would suggest `"standalone"` and `ModelResourceLocation.standalone(...)`.